src/lib/forward.ts

```diff
@@ -333,13 +363,24 @@ export class EventForwarder {
                 delete args.dataTypeLabel;
               }
             }
-            this.sendEvent(client, {
-              source: "node",
-              event: "notification",
-              nodeId: changedNode.nodeId,
-              ccId,
-              args,
-            });
+            if (client.schemaVersion < 32) {
+              this.sendEvent(client, {
+                source: "node",
+                event: "notification",
+                nodeId: changedNode.nodeId,
+                ccId,
+                args,
+              });
```

Comment:
Do the clients have validation for extra properties? If not, we could simply pass the endpoint index independent of the schema version.

Reply:
they may. We had originally intended to do that for HA but couldn't with the validator we were using